### DISCUSSION TITLE
replica from MS SQL 2014

### DISCUSSION THREAD

**Discussion by Gyorgy-Szy** [planetscale/discussion#534] [Q&A] [answered]
Hi, I need to create a live read only replica from a MS SQL 2014 server into the planetscale database, near real time (max 10 mins of latency).
Is it possible somehow natively (or what third part tools may I need for this?)
Thanks!

**Comment by orware** [comment, marked as answer]
Hello @Gyorgy-Szy,

Since you also submitted a ticket for this same question, I hope the answer I was able to share with you yesterday proves to be helpful for your needs and we can continue the discussion there in your ticket if needed!

**Reply by Gyorgy-Szy** [reply]
Yes, thank you for your help! I have started to develop a solution following your idea! Thanks!
